back-end/API/app/models/user_model.py

```
@@ -8,11 +8,11 @@ class User(db.Model):
     password = db.Column(db.String(300), default=False)
     active = db.Column(db.Boolean, default=False)
     isModerator = db.Column(db.Boolean, default=False)
+    #Enventuellement, un employeur vérifié pourrait publier des offres d'emplois sans passer par le processus d'approbation.
+    verified = db.Column(db.Boolean, nullable=False , default=False)
+    # Relations
+    enterpriseId = db.Column(db.Integer, db.ForeignKey("enterprise.id"), nullable=True)
+    enterprise = db.relationship("Enterprise", back_populates="users")
 
     def __repr__(self):
```

Ruff (0.15.15)

[warning] Missing return type annotation for special method `__repr__`

Add return type annotation: `str`

(ANN204)